src/modules/missionHelper/i18n/de_DE.json

```diff
@@ -158,11 +158,21 @@
             "allow_ktw_instead_of_rtw": {
                 "ambulances": "RTW oder KTW oder KTW Typ B"
             },
+            "allow_police_motorcycle_instead_of_fustw": {
```

Comment:
We don't need the translations doubled. If needed without the module active => de_DE.root.json. If only needed with module active => de_DE.json

Reply:
please explain this bug

Reply:
Nvm, I just compared the objects by title and not by the key-pairs.